utils/convert.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import (
+	"encoding/base64"
+	"fmt"
+)
+
+const (
+	Text = iota + 1
+	Base64
+)
+
+func StringToBytes(str string, encoding int) ([]byte, error) {
```

Comment:
Shouldn't this belong to go-common repo?
similar to https://github.com/TRON-US/go-common/blob/master/operator/string.go

Reply:
It is a little btfs-offline-related by defining custom encoding/decoding schema

Reply:
But yeah, can be general.